zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/DocumentenApiPlugin.kt

```diff
@@ -428,6 +439,16 @@ class DocumentenApiPlugin(
         }
     }
 
+    private fun setDocumentUrlProcessVariableForRelatedProcess(
+        relatedDocumentId: String,
+        documentUrl: String
+    ) {
+        val relatedProcessDocument = processDocumentAssociationService.findProcessDocumentInstances(JsonSchemaDocumentId.existingId(relatedDocumentId)).firstOrNull() ?: return;
```

Comment:
This won't work of it's an upload in a supporting or sub process. Should we also include the process instance id?